webstepper/src/App.tsx

```diff
@@ -8,42 +8,56 @@ import "./css/styles.css";
 export default function App() {
     const [step, setStep] = useState<number>(0);
     // TODO: replace this with actual code to display
-    const codeText = `num = 123
-some_string = "Hello, world"
-num2 = 321
-arr = [some_string, "string 123321"]`;
-    const limit = codeText.split("\n").length;
+    const codeText = `num = [1, 2, 3]
+for i in range(len(nums)):
+    if i == 0:
+        nums[i] = nums[i] + 1
+    else:
+        nums[i] = nums[i] * 2
+`;
+    const limit = Object.keys(window.svgArray).length;
     const handleStep = (newStep: number) => {
         setStep(Math.min(Math.max(newStep, 0), limit - 1));
     };
-    const svgPath = `/images/snapshot-${step}.svg`;
 
     return (
         <main className="container">
             <Header />
             <Stack direction="row" spacing={2}>
                 <Box sx={{ width: "40%" }}>
                     <h2>Code</h2>
-                    <Typography>Line: {step + 1}</Typography>
+                    <Typography>
+                        Line: {window.svgArray[step].lineNumber}
+                    </Typography>
```

Comment:
Let's replace this with something like Step <step>/<total steps> . I think that will be more informative